src/baseCommand.ts

```
@@ -0,0 +1,68 @@
+import { debuglog } from 'node:util';
+import { Command, Flags, Interfaces } from '@oclif/core';
+import { PackageEgg } from './types.js';
+import { readJSON } from 'utility';
+import path from 'node:path';
+
+const debug = debuglog('@eggjs/scripts/baseCommand');
+
+type Flags<T extends typeof Command> = Interfaces.InferredFlags<typeof BaseCommand['baseFlags'] & T['flags']>;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Flags'. Consider to delete it or rename it.

'Flags' is defined here:


(lint/suspicious/noRedeclare)